lib/restricted-input.js

```diff
@@ -26,7 +26,16 @@ function RestrictedInput(options) {
     throw new Error(constants.errors.PATTERN_MISSING);
   }
 
-  if (device.isIos()) {
+  // SamsungBrowser contains 'Chrome' in userAgent, so check it first
+  if (device.isSamsungBrowser()) {
+    // Disable formatting due to various digits being dropped
+    this.strategy = {
```

Comment:
I don't think so, because we don't actually want any of the behavior of the base strategy.